bpfd-api/src/util.rs

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: (MIT OR Apache-2.0)
+// Copyright Authors of bpfd
+
+#[rustfmt::skip]
+pub mod directories {
+    // The following directories are used by bpfd. They should be created by bpfd service
+    // via the bpfd.service settings. They will be manually created below in case bpfd
+    // is not being run as a service.
+    //
+    // ConfigurationDirectory: /etc/bpfd/
+    pub const CFGDIR:                 &str = "/etc/bpfd";
+    pub const CFGDIR_CA_CERTS:        &str = "/etc/bpfd/certs/ca";
+    pub const CFGDIR_STATIC_PROGRAMS: &str = "/etc/bpfd/programs.d";
+    pub const CFGPATH_BPFD_CONFIG:    &str = "/etc/bpfd/bpfd.toml";
+    //
+    // RuntimeDirectory: /run/bpfd/
```

Comment:
What was the reason to move from `/var/run` again?

Reply:
Systemd manages `/run/`, not `/var/run/`. On most systems, `/var/run/` is a symlink to `/run/` so it is a noop, but this is not guaranteed.